modules/account-lib/src/coin/hbar/utils.ts

```diff
@@ -0,0 +1,42 @@
+import * as _ from 'lodash';
+
+import { AccountId } from '@hashgraph/sdk/lib/account/AccountId';
+import { TransactionId } from '@hashgraph/sdk';
+
+/**
+ * Returns whether or not the string is a valid Hedera account.
+ *
+ * In any form, `shard` and `realm` are assumed to be 0 if not provided.
+ *
+ * @param {string} address - the  address to be validated
+ * @returns {boolean} - the validation result
+ */
+export function isValidAddress(address: string): boolean {
+  if (_.isEmpty(address)) {
+    return false;
+  }
+  try {
+    const acc = AccountId.fromString(address);
+    return !_.isNaN(acc.account);
+  } catch (e) {
+    return false;
+  }
+}
+
+/**
+ * Returns whether or not the string is a valid Hedera transaction id or not
+ *
+ * @param {string} txId - the transaction id to be validated
+ * @returns {boolean} - the validation result
+ */
+export function isValidTransactionHash(txId: string): boolean {
```

Comment:
In Hedera, transactions have an `id` and a `hash`: https://docs.hedera.com/guides/docs/sdks/transactions#transaction-record.

I think we should call this function: `isValidTransactionId()`, to avoid confusion.

Reply:
I agree. However, I would like to change it for other coins too to be consistent.
I'll add that as a follow-up task because it will require some updates in other systems consuming it.